modules/vaos/spec/requests/vaos/v2/appointments_spec.rb

```diff
@@ -1555,6 +1555,44 @@ def stub_clinics
         end
       end
 
+      context 'when the upstream service returns a 500 error' do
+        it 'returns a bad_gateway status and appropriate error message' do
+          VCR.use_cassette('vaos/eps/get_appointments/500_error') do
+            VCR.use_cassette('vaos/v2/appointments/get_appointments_200') do
+              VCR.use_cassette('vaos/eps/get_drive_times/200') do
+                VCR.use_cassette 'vaos/eps/get_provider_slots/200' do
+                  VCR.use_cassette 'vaos/eps/get_provider_service/200' do
+                    VCR.use_cassette 'vaos/eps/draft_appointment/200' do
+                      VCR.use_cassette 'vaos/eps/token/token_200' do
```

Comment:
😳

Reply:
@rmtolmach can you elaborate? I know this is quite a bit of nesting of cassettes, but the process runs quite a few external calls, is there something specific you'd like changed?